### PR TITLE
WebExtension: Added Check for Whether Canonical Doc Links to AMP

### PR DESCRIPTION
**Pull request by tomayac**
# WebExtension: Add Check for Whether Canonical Doc Links to AMP Doc

Sometimes developers forget to link to the AMP document from the canonical document.
This Pull Request addresses this issue and displays a warning if this is the case:

screen shot 2017-10-25 at 17 12 09

### CODE REVIEW

File: validator/webextension/_locales/en/messages.json [hunk 1]
```diff
@@ -48,5 +48,14 @@
         "content": "$1"
       }
     }
+  },
+  "warningCanonicalDoesntLinkToAMP": {
```
Comment (honeybadgerdontcare) [inline validator/webextension/_locales/en/messages.json]: nit: change Doesnt to DoesNot

Comment (tomayac) [reply validator/webextension/_locales/en/messages.json]: Addressed with https://github.com/ampproject/amphtml/pull/11809/commits/a4656bf26f90d75e40ef02102d2bdbd8ce18a4b8.

**Comment by honeybadgerdontcare** [comment]
@tomayac conflicts still remain

**Comment by tomayac** [comment]
Call me stupid, but I don't see the conflict. In the ```manifest.json``` file, I have bumped the version number to 1.1.4 and recall I had strange invisible spaces that I have replaced with regular spaces ([diff](https://github.com/ampproject/amphtml/pull/11809/commits/e453e01c01fc5f543d674199281a618402993ddb#diff-d6bba31408c15049d512538c46e0b953L4)). The file looks like [this](https://github.com/tomayac/amphtml/blob/b60ccec61a9daa83840976f92c52df408bed7ef5/validator/webextension/manifest.json) for me now.

**Comment by honeybadgerdontcare** [comment]
@tomayac lets just remove the version update and do that in a separate PR.

**Comment by tomayac** [comment]
@honeybadgerdontcare Any update on this? I was thinking, maybe the validator itself should run this check, and not just the extension?! What do you think? But then it's not strictly *invalid* AMP, just *non-discoverable* AMP.

**Comment by honeybadgerdontcare** [comment]
@tomayac This looks great, lets get it in. Unfortunately we just moved amp-validator.html to popup-validator.html, so you'll need to move your code to that instead.

**Comment by aghassemi** [comment]
Thanks for the pull request. Since this PR has been stale for a while, I am closing it. Please feel free to reopen as needed.

**Pull request closed without merging**